### DISCUSSION TITLE
Metadata-related questions

### DISCUSSION THREAD

**Discussion by Yibing-Du** [ohtap/ohtap#39] [General]
I'm starting this since I might encounter more issues as I try to match NVivo output with metadata (both interviews and interviewees), and I thought here would be easier than long email threads. @ebf77 @njmarine 

I just noticed the "is_in_rape_cluster" and "is_in_abortion_cluster" columns for interviews. How are they determined? Are they up-to-date?

**Comment by Yibing-Du** [comment]
I'm so sorry if these questions have been addressed before/elsewhere (and if there's some documentation that I missed, please kindly let me know!), but another question here: What does RBNF 1 mean for marriage/children status?

**Reply by ebf77** [reply]
good idea to start this forum. RBNF1 = Researched But Not Found (once). RBNF2= second try. After 2 tries we stopped. However, we tried to complete the searches only for the priority metadata categories, so I suggest limiting your efforts to those: birth year/cohort, interview year, race, and education. We have not been using marriage/children status.

**Comment by Yibing-Du** [comment]
@ebf77 @njmarine 

I'm encountering strange issues when I merge metadata with NVivo extracts. I haven't figured it out yet, but meanwhile I've 1. deduplicated the texts (still some minor issues), 2. gotten an "any_node" for each person with any_mention (about 500 people in total), and 3. put together a metadata file that combines information from both the interviewee page and the interview page (I'm possibly re-inventing the wheel here, but hopefully not). 

[without_metadata_all_extract_0323.xlsx](https://github.com/ohtap/ohtap/files/6194980/without_metadata_all_extract_0323.xlsx)

For 1: The "Coded Text" column is just for my own reference, so it should be ignored. The actual text column is “corrected_text.”
There're cases of needs review (there's a column for it) that I think someone can quickly resolve. I still need to (1) clean the texts, (2) put them back to the original order, and (3) resolve some weird cases when two texts that overlap are still on separate rows (only a few). 

[without_metadata_node_any.xlsx](https://github.com/ohtap/ohtap/files/6194981/without_metadata_node_any.xlsx)

For 2: I'll run regression on these nodes too, once I match them with metadata. 

[merged_metadata_cleaned_0323.xlsx](https://github.com/ohtap/ohtap/files/6194978/merged_metadata_cleaned_0323.xlsx)

For 3: I kept a few more columns than what we typically use (ones that we have the info for at least ~2/3 of all female interviewees). This is definitely lower priority, but I think birth state can help us with sanity-check (together with collection id), and the year of the first interview is something that's mentioned in today's meeting, so anyone who needs that information can conveniently access it here. 

I'm probably going to be busy in the following weeks, so I'm not entirely sure if I can resolve the issues before our next meeting. I'll spend time on regression instead. But please comment, give suggestions, & use these sheets if they're relevant to folks!

**Comment by ebf77** [comment]
Quick question: is Jade getting this thread?
I thought she had merged metadata and NVivo. How does this task differ? Share with her. Thanks
Estelle
Sent from my iPhone

On Mar 23, 2021, at 11:54 PM, Yibing-Du ***@***.***> wrote:

﻿

@ebf77<https://github.com/ebf77> @njmarine<https://github.com/njmarine>

I'm encountering strange issues when I merge metadata with NVivo extracts. I haven't figured it out yet, but meanwhile I've 1. deduplicated the texts (still some minor issues), 2. gotten an "any_node" for each person with any_mention (about 500 people in total), and 3. put together a metadata file that combines information from both the interviewee page and the interview page (I'm possibly re-inventing the wheel here, but hopefully not).

without_metadata_all_extract_0323.xlsx<https://github.com/ohtap/ohtap/files/6194980/without_metadata_all_extract_0323.xlsx>

For 1: The "Coded Text" column is just for my own reference, so it should be ignored. The actual text column is “corrected_text.”
There're cases of needs review (there's a column for it) that I think someone can quickly resolve. I still need to (1) clean the texts, (2) put them back to the original order, and (3) resolve some weird cases when two texts that overlap are still on separate rows (only a few).

without_metadata_node_any.xlsx<https://github.com/ohtap/ohtap/files/6194981/without_metadata_node_any.xlsx>

For 2: I'll run regression on these nodes too, once I match them with metadata.

merged_metadata_cleaned_0323.xlsx<https://github.com/ohtap/ohtap/files/6194978/merged_metadata_cleaned_0323.xlsx>

For 3: I kept a few more columns than what we typically use (ones that we have the info for at least ~2/3 of all female interviewees). This is definitely lower priority, but I think birth state can help us with sanity-check (together with collection id), and the year of the first interview is something that's mentioned in today's meeting, so anyone who needs that information can conveniently access it here.

I'm probably going to be busy in the following weeks, so I'm not entirely sure if I can resolve the issues before our next meeting. I'll spend time on regression instead. But please comment, give suggestions, & use these sheets if they're relevant to folks!
